helm/kubeless/values.yaml

```diff
@@ -0,0 +1,52 @@
+# Default values for kubeless.
+
+## Controller configuration
+controller:
+
+  deployment:
+    replicaCount: 1
+    image:
+      repository: bitnami/kubeless-controller
+      tag: 0.0.12
```

Comment:
should we use sha like kafka and zookeeper? I personally prefer using tags everywhere.